cms/static/cms/js/modules/shortcuts/placeholders.js

```diff
@@ -0,0 +1,134 @@
+var keyboard = require('keyboardjs');
+var $ = require('jquery');
+
+require('../jquery.trap');
+
+var bindPlaceholderKeys = function () {
```

Comment:
would be nice to have some comments describing what actions are bound, like ``registers [escape] shortcut for closing the structure focus``